web/src/Components/Combinations/CombinationCard.tsx

```diff
@@ -95,6 +96,10 @@ export const CombinationCard = ({
 
     CombinationAPI.postCombinationApi(token, Object.values(combination.values))
       .then((response) => {
+        if (!response.data) {
+          ErrorToast('No data received from API', 500)
+          return
+        }
```

Comment:
This looks strange.
500 API responses should raise an error, not return null data
And should be caught by .then().catch(...)